apps/react-18-tests-v8/jest.config.js

```diff
@@ -9,12 +12,18 @@ module.exports = {
   globals: {
     'ts-jest': {
       tsConfig: '<rootDir>/tsconfig.spec.json',
-      diagnostics: false,
+      diagnostics: { warnOnly: true, exclude: ['packages/**'] },
```

Comment:
this exposed lot of warnings from v8 packages - caused by not following isolateModules. 

this will resolve it https://github.com/microsoft/fluentui/pull/25774